hw_3/hw_3/ViewController.swift

```diff
@@ -0,0 +1,177 @@
+        print("\(firstnumber) + \(secondNumber) = \(numberSum) ")
+    }
+    
+    func summa(_ firstnumber: Double,_  secondNumber: Int) {
+        let numberSum = firstnumber + Double(secondNumber)
+        print("\(firstnumber) + \(secondNumber) = \(numberSum) ")
+    }
+    
+    func summa(_ firstnumber: Double,_  secondNumber: Double) {
+        let numberSum = firstnumber + secondNumber
+        print("\(firstnumber) + \(secondNumber) = \(numberSum) ")
+    }
+    
+    func subtraction(_ firstnumber: Int,_  secondNumber: Int) {
+        let numberSum = firstnumber - secondNumber
+        print("\(firstnumber) - \(secondNumber) = \(numberSum) ")
+    }
+    
+    func subtraction(_ firstnumber: Int,_  secondNumber: Double) {
+        let numberSum = Double(firstnumber) - secondNumber
+        print("\(firstnumber) - \(secondNumber) = \(numberSum) ")
+    }
+    
+    func subtraction(_ firstnumber: Double,_  secondNumber: Int) {
+        let numberSum = firstnumber - Double(secondNumber)
+        print("\(firstnumber) - \(secondNumber) = \(numberSum) ")
+    }
+    
+    func subtraction(_ firstnumber: Double,_  secondNumber: Double) {
+        let numberSum = firstnumber - secondNumber
+        print("\(firstnumber) - \(secondNumber) = \(numberSum) ")
+    }
+    
+    func multiply(_ firstnumber: Int,_  secondNumber: Int) {
+        let numberSum = firstnumber * secondNumber
+        print("\(firstnumber) * \(secondNumber) = \(numberSum) ")
+    }
+    
+    func multiply(_ firstnumber: Int,_  secondNumber: Double) {
+        let numberSum = Double(firstnumber) * secondNumber
+        print("\(firstnumber) * \(secondNumber) = \(numberSum) ")
+    }
+    
+    func multiply(_ firstnumber: Double,_  secondNumber: Int) {
+        let numberSum = firstnumber * Double(secondNumber)
+        print("\(firstnumber) * \(secondNumber) = \(numberSum) ")
+    }
+    
+    func multiply(_ firstnumber: Double,_  secondNumber: Double) {
+        let numberSum = firstnumber * secondNumber
+        print("\(firstnumber) * \(secondNumber) = \(numberSum) ")
+    }
+    
+    func devision(_ firstnumber: Int,_  secondNumber: Int) {
+        let numberSum = Double(firstnumber) / Double(secondNumber)
+        print("\(firstnumber) / \(secondNumber) = \(numberSum) ")
+    }
+    
+    func devision(_ firstnumber: Int,_  secondNumber: Double) {
+        let numberSum = Double(firstnumber) / secondNumber
+        print("\(firstnumber) / \(secondNumber) = \(numberSum) ")
+    }
+    
+    func devision(_ firstnumber: Double,_  secondNumber: Int) {
+        let numberSum = firstnumber / Double(secondNumber)
+        print("\(firstnumber) / \(secondNumber) = \(numberSum) ")
+    }
+    
+    func devision(_ firstnumber: Double,_  secondNumber: Double) {
+        let numberSum = firstnumber / secondNumber
+        print("\(firstnumber) / \(secondNumber) = \(numberSum) ")
+    }
+
+    func stringComparison(firstString: String, secondString: String) {
```

Comment:
It's better to have a verb in the function name. For example, you can rename `stringComparison` to `compareStrings`